lib/membrane_mp4/payloader/aac.ex

```diff
@@ -40,7 +40,7 @@ defmodule Membrane.MP4.Payloader.AAC do
   @impl true
   def handle_process(:input, buffer, _ctx, state) do
     # we set DTS=PTS, as ISO base media file format specification uses DTS for calculating deltas
-    buffer = %Buffer{buffer | dts: buffer.pts}
+    buffer = %Buffer{buffer | dts: Buffer.get_dts_or_pts(buffer)}
```

Comment:
Is the comment above still valid with this change?

Reply:
I'll rephrase it a little bit